src/main/scala/exercises/06_Collections.scala

```diff
@@ -114,7 +114,12 @@ object Collections {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList match {
+      case e1::e2::tail   => e1 :: oddElements(tail)
+      case e1::Nil        => List(e1)
```

Comment:
This again looks a lot cleaner with the `::` idiom. I also didn't realize that there was `Nil`. This definitely looks way better.